### ISSUE TITLE
 [Bug] - Permissions in subfolders are not applied to upper folders

### ISSUE THREAD

**Issue by wzrdtales** [ONLYOFFICE/CommunityServer#431]
When files in the common section are shared, the permissions have to be given from the bottom up. If you want to give a person specifically access to only one folder inside one folder, this is impossible, since that means you need to give access to the bottom folder and then change it in **all** other folders inside...

**Comment by HelenBelova17** [comment]
Hello @wzrdtales,
Currently, we have the following hierarchy: If you do not specify specific permissions for a subfolder, the subfolder inherits the permissions of the parent folder.
If certain permissions are set for a subfolder, the permissions for the subfolder have a higher priority than the permissions for its parent folder. More information is available [here](https://helpcenter.onlyoffice.com/userguides/groups-tipstricks-documents-folder-permissions.aspx).
Or specify a detailed scenario in which these permissions do not work.

**Comment by wzrdtales** [comment]
> If certain permissions are set for a subfolder, the permissions for the subfolder have a higher priority than the permissions for its parent folder.

@HelenBelova17 This is wrong then unfortunately. Exactly this does not work. If the parent folder (for example root level in the common space) has explicitly set to prevent everyone seeing it by default and you set then a subfolder with viewing permissions, the parent folder wont show up at all.

**Comment by wzrdtales** [comment]
> if you set the filter.

which filter? SInce this is tracked, is that already planned or no ETA yet?

**Comment by wzrdtales** [comment]
that is unfortunate since this just breaks the "common" storage section completely, there is not really a case where everything is shared there with everyone.

Also is it normal that shared documents in common do not appear in the shared with me section?

**Comment by HelenBelova17** [comment]
> that is unfortunate since this just breaks the "common" storage section completely, there is not really a case where everything is shared there with everyone.
> 
> Also is it normal that shared documents in common do not appear in the shared with me section?

Yes, it is normal.
